src/directives/threema_action.ts

```diff
@@ -15,22 +15,38 @@
  * along with Threema Web. If not, see <http://www.gnu.org/licenses/>.
  */
 import {UriService} from '../services/uri';
+import {WebClientService} from '../services/webclient';
 
 export default [
     '$timeout',
     '$state',
     'UriService',
-    function($timeout, $state: ng.ui.IStateService, uriService: UriService) {
+    'WebClientService',
+    function($timeout, $state: ng.ui.IStateService, uriService: UriService, webClientService: WebClientService) {
```

Comment:
Maybe this directive should be renamed to `click_action` now that it does not just handle threema actions?

Reply:
+1

Reply:
done